src/Connection.cpp

```diff
@@ -254,8 +252,15 @@ connectionGeometry()
   return _connectionGeometry;
 }
 
+ConnectionGeometry const&
```

Comment:
For `ConnectionPainter::paint` which now takes in a `Connection const&` so there needs to be const access to the geometry